src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/patch/fabric/FabricMinecraftTransformer.kt

```diff
@@ -146,4 +148,23 @@ abstract class FabricMinecraftTransformer(
             }
         }
     }
+
+    override fun configureRemapJar(task: AbstractRemapJarTask) {
+        task.manifest {
+            it.attributes(mapOf(
+                "Fabric-Minecraft-Version" to provider.version,
+                "Fabric-Loader-Version" to fabricDep.version,
+            ))
+        }
+        if (fabricDep.version?.let { SemVerUtils.matches(it, ">=0.15.0") } == true) {
+            project.logger.info("enabling mixin extra")
+            if (task is RemapJarTask) {
+                task.mixinRemap {
+                    enableMixinExtra()
+                }
+            }
+        }
+    }
+
+    open fun additionalRemapJarConfiguration(task: AbstractRemapJarTask) {}
```

Comment:
I'm not seeing this function called. Also the children could just call super on `configureRemapJar`

Reply:
OrnitheMinecraftTransformer extends LegacyFabricMinecraftTransformer so calling super won't work in this case. Or it has to be handled differently.